pandas/tests/reshape/test_pivot.py

```diff
@@ -906,6 +937,7 @@ def test_categorical_margins(self):
 
         data = df.copy()
         table = data.pivot_table('x', 'y', 'z', margins=True)
+        pytest.xfail('GH 17035 (np.mean of ints is casted back to ints)')
```

Comment:
Just for reference, these are only temporary until subsequent bugs are patched (same for the one you added above)?